api/src/certification/shared/domain/models/CertificationCourse.js

```diff
@@ -144,6 +147,10 @@ class CertificationCourse {
     return this._isRejectedForFraud;
   }
 
+  adjustForAccessibility(isAdjustmentNeeded) {
+    this._isAdjustedForAccessibility = isAdjustmentNeeded;
```

Comment:
picky winky : renforcer le setter en l'abscence total de control par JS. Je propose que ca soit fait aussi dans le contrsctutor.

```suggestion
    this._isAdjustedForAccessibility = !!isAdjustmentNeeded;
```

Reply:
(D'ailleurs ca resolvera tous tes cas de undefined 😉